CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ You can now pass `--cdn` to the gulp task to bundle assets pointing towards the
 * `Dropdown`: Options list is always rendered to the DOM, but is hidden until selected
 * `Textarea` now accepts a new prop `warnOverLimit` to display the character count message in red.
 * Simplify character count in `Textarea`.
+* `Form` now has default `SaveButton` and `CancelButton` functional stateless componenents. The former can be overriden with a new prop of `customSaveButton`.
```

Comment:
Changelog file has moved along since this PR was raised. Needs moving up the document